frontend/src/components/pages/acls/Acl.List.tsx

```diff
@@ -593,24 +605,88 @@ const RolesTab = observer(() => {
   );
 });
 
-const AclsTab = observer((p: { principalGroups: AclPrincipalGroup[] }) => {
-  useEffect(() => {
-    void api.refreshAcls(AclRequestDefault, true);
-  }, []);
+interface SimpleAcl {
+  host: string;
+  principal: string;
+  principalType: string;
+  principalName: string;
+  hasAcl: boolean;
+}
+
+const AclsTab = observer((_: { principalGroups: AclPrincipalGroup[] }) => {
+  const { data: principalGroups, isLoading } = useQuery(listACLs, {} as ListACLsRequest, {
+    select: (response) => {
+      const groupsAcl = response.resources.reduce((acc, r) => {
+        r.acls.forEach((a) => {
+          if (!acc.has(`${a.principal}:${a.host}`)) {
+            acc.set(`${a.principal}:${a.host}`, {
+              host: a.host,
+              principal: a.principal,
+              principalType: a.principal.split(':')[0] || '',
+              principalName: a.principal.split(':')[1] || '',
+              hasAcl: true,
+            });
+          }
+        });
+        return acc;
+      }, new Map<string, SimpleAcl>());
+      return groupsAcl.values().toArray();
+    },
+  });
+
+  const { mutateAsync: deleteACLMutation } = useMutation(deleteACLs);
```

Comment:
Could be a `useDeleteACLsMutation` custom hook with `onSuccess`/`onError` to invalidate `queryClient` queries.